typings/index.d.ts

```diff
@@ -1868,7 +1868,7 @@ declare module 'discord.js' {
     public lastMessageID: Snowflake | null;
     public readonly partial: false;
     public readonly presence: Presence;
-    public system: boolean | null;
+    public system: boolean;
```

Comment:
Can't it still be null?

Reply:
Only if the object is a partial, otherwise it will always be a boolean

Reply:
Shouldn't this be `boolean` in jsdoc too? There it's still `?boolean`.

Reply:
No, because the JSDoc can’t tell if the object is a partial or not, and the typings can